doc/man7/password-handling.pod

```diff
@@ -0,0 +1,89 @@
+=pod
+
+=head1 NAME
+
+password handling - How diverse parts of OpenSSL treat pass phrases
+
+=head1 DESCRIPTION
+
+=head2 The general case
+
+In a modern world with all sorts of character encodings, the treatment of pass
+phrases has become increasingly complex.
```

Comment:
you could add something like:

> This manual page attempts to give an overview over how this problem is currently addressed in different parts of the openssl library.

and the proceed with 'The general case'.

Reply:
Thank you